### ISSUE TITLE
github actions for unittesting

### ISSUE THREAD

**Issue by quaat** [SINTEF/soft7#2]
Setup github actions for unittesting

**Comment by quaat** [comment]
Add support for our standard linters

**Comment by CasperWA** [comment]
First we need actual tests ;)

**Comment by CasperWA** [comment]
Blocked by #11

**Comment by CasperWA** [comment]
This is now present in the `develop` branch (which will eventually be merged into the default `main` branch).
